student-work/2022/Debbie-Mankowitz/api-final-project/README.md

```diff
@@ -0,0 +1,25 @@
+# Hamburger API DOC
+This is the new APP for customers for online orders for General Putnam Motel Diner (GPMD). 
```

Comment:
```suggestion
This API is for creating a new APP for General Putnam Motel Diner (GPMD).  Customers of GPMD can use the app to order food. This initial offering is a Proof of Concept (PoC). 
```

Reply:
Debbie if you like the change I am suggesting click the "Commit Suggestion" button and commit the change.